### PR TITLE
Update MobileApp endpoint

### PR DESCRIPTION
**Pull request by nicholaspad**
The MobileApp team recently updated the API and changed the version to 1.0.2 (originally 1.0.0). This change broke TigerSnatch, a new TigerApp that relies on MobileApp. The base endpoint must now be either `https://api.princeton.edu/mobile-app/1.0.2` or `https://api.princeton.edu/mobile-app`. This PR uses the latter since it's guaranteed to be up-to-date. **See the support ticket below for more details.** Verified that the new URL works via an API request by TigerSnatch.

image

### CODE REVIEW

**Review by vr2amesh — APPROVED** [review]
Thanks for making this change! Feel free to merge your request into the master branch.

**Comment by nicholaspad** [comment]
No problem! I actually can't merge it because I don't have write access. Can you do it?

**Pull request merged**
